README.md

```diff
@@ -43,6 +43,24 @@ You can run Companienv automatically after `composer install` or `composer updat
 }
 ```
 
+The ``.env`` will then be created or updated by the composer script, to match the structure of the dist file .env.dist by asking you the missing parameters.
```

Comment:
I guess you could remove this sentence, it's a bit duplicated from the other context I believe.